torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -105,13 +105,13 @@ struct ConstantPropagator {
   // Runs constant propagation with an aliasing db and checks if inputs or
   // outputs might be mutated in the graph
   static ConstantPropagator WithAliasDb(std::shared_ptr<Graph> graph) {
```

Comment:
OOC, why are these `shared_ptr` not const ref?

Reply:
There is transfer of ownership here, so I think value type + `std::move` makes more sense.